recipes-devtools/python/python3-labgrid/labgrid-coordinator.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Labgrid Coordinator
+After=network.target
+
+[Service]
+Environment="PYTHONUNBUFFERED=1"
+Environment="LABGRID_COORDINATOR_LISTEN_IP=0.0.0.0"
+Environment="LABGRID_COORDINATOR_PORT=20408"
+EnvironmentFile=/etc/labgrid/environment
+ExecStart=/usr/bin/labgrid-coordinator -l ${LABGRID_COORDINATOR_LISTEN_IP}:${LABGRID_COORDINATOR_PORT}
+Restart=on-failure
+DynamicUser=yes
```

Comment:
I'm not sure that's a good idea. The state directory permissions will probably cause issues after an update.